docs/docker/README.md

```diff
@@ -55,7 +55,7 @@ docker run \
 
 ## Run Distributed MinIO on Docker
 
-Distributed MinIO can be deployed via [Docker Compose](https://docs.min.io/docs/deploy-minio-on-docker-compose). This means Docker Compose lets you quickly get started with Distributed MinIO on your computer - ideal for development, testing, staging environments. We recommend kubernetes based deployment for production level deployment <https://github.com/minio/operator>.
+Distributed MinIO can be deployed via [Docker Compose](https://min.io/docs/minio/container/operations/install-deploy-manage/deploy-minio-single-node-multi-drive.html). This means Docker Compose lets you quickly get started with Distributed MinIO on your computer - ideal for development, testing, staging environments. We recommend kubernetes based deployment for production level deployment <https://github.com/minio/operator>.
```

Comment:
So we do not actually have a multi-node multi-drive instruction for Docker, and have at this time dropped the compose docs.

Given the challenge of networking, it might be better to drop this section entirely and just note "For distributed deployments in an orchestrated environment, use the MinIO Kubernetes Operator"

Reply:
Removed the references to Docker and left the existing recommendation for Kubernetes for distributed deployments. I added a link to the Kubernetes root page, as well.